packages/manager/apps/container/src/container/nav-reshuffle/header/user-account-menu/style.module.scss

```diff
@@ -31,6 +31,10 @@
   user-select: text;
 }
 
+.ssoBadge {
```

Comment:
**question:** Couldn't the helper `ml-0` be used? (we tend to avoid custom style)

Reply:
Yes I can, done :)